pkg/enum/linear.go

```
@@ -0,0 +1,660 @@
+			"title":      title,
+			"url":        url,
+			"team":       team,
+			"project":    project,
+		},
+	}
+}
+
+// logf writes a progress message when verbose output is enabled.
+func (e *LinearEnumerator) logf(format string, args ...interface{}) {
+	if e.config.Verbose != nil {
+		fmt.Fprintf(e.config.Verbose, format+"\n", args...)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)